packages/storage-plus/src/indexed_map.rs

```diff
@@ -1,12 +1,9 @@
-// this module requires iterator to be useful at all
-#![cfg(feature = "iterator")]
```

Comment:
Why remove iterator here?

Do we run tests without iterator feature? We should...

Reply:
Secret doesn't allow iterator, and we are working that their 1.0 contracts are compatible with other networks.

Reply:
OK, sorry about that. I thought iterator was gone as a feature. Will revert, which will cost me a little rebasing. :-)

Reply:
Reverted the iterator changes. Let's tackle any failing tests in a different PR.